whiteluna/acsylla
# Some improvements (#48)

* Add support for all native types and add method as_dict to row object.
native_type: ASCII
           : | BIGINT
           : | BLOB
           : | BOOLEAN
           : | COUNTER
           : | DATE
           : | DECIMAL
           : | DOUBLE
           : | DURATION
           : | FLOAT
           : | INET
           : | INT
           : | SMALLINT
           : | TEXT
           : | TIME
           : | TIMESTAMP
           : | TIMEUUID
           : | TINYINT
           : | UUID
           : | VARCHAR
           : | VARINT

* Add support for collections types, tuple and udt

* Add support for all scylla types and add method Row.as_dict()

* Add batch counter

* delete old repo

* add scylladb/cpp-driver submodule

* Use https://github.com/scylladb/cpp-driver

* fix types

* use last commit for cpp-driver

* change cpp-driver url

* Add build script

* fix warning for type comparison

* install latest cython

* Added suport for all types

* return default value for --duration

* return version a0

* try to fix build in github CI

* Use BRANCH from command args

* Remove unused commented code

* Revert update submodule

* Change connection port for default 9042

* updating submodule to latest

* run make format and make lint

* run make format

* Exclude vendor directory for black check

* Update ci.yml

Change sleep time for wait till cassandra is up
